PowerEditor/installer/nsisInclude/gobalDef.nsh

```diff
@@ -34,6 +34,12 @@
 !define VERSION_MAJOR 7
 !define VERSION_MINOR 0
 
+!define CompanyName		"Don HO don.h@free.fr"
+!define Description		"Notepad++ : a free (GNU) source code editor"
+!define Version 		"7.0.0.0"
```

Comment:
If you are talking about existing macro APPVERSION, it can't be used directly as VIProductVersion expect version string X.X.X.X. We have to write many conditions to utilize APPVERSION. 
Reason is:
In V7 APPVERSION = 7
In V6.9.2 APPVERSION = 6.9.2
In V6.9 APPVERSION = 6.9